app/models/login.php

```diff
@@ -0,0 +1,42 @@
+<?php
+class Login extends AppModel
+{
+    const MIN_STRING_LENGTH = 1;
```

Comment:
Please make the name easy to understand. (e.g. MIN_USERNAME_LENGTH)
since it is used only  for username validation.